docker-compose.yml

```diff
@@ -30,6 +30,7 @@ services:
       - "polis-net"
     ports:
       - "5000:5000"
+      - "9229:9229"
```

Comment:
Is this a vscode thing? Maybe document IDE stuff somewhere?

Reply:
This should be removed from here, since it's already in `docker-compose.dev.yml`; We do not want random ports open in prod.